api/src/main/java/org/openmrs/module/metrics/builder/impl/JmxReportBuilderImpl.java

```diff
@@ -0,0 +1,31 @@
+package org.openmrs.module.metrics.builder.impl;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.jmx.JmxReporter;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.module.metrics.builder.JmxReportBuilder;
+
+public class JmxReportBuilderImpl implements JmxReportBuilder {
+	
+	protected final Log log = LogFactory.getLog(this.getClass());
+	
+	private static MetricRegistry metricRegistry = new MetricRegistry();
+	
+	@Override
+	public MetricRegistry initializeMetricRegistry() {
+		return metricRegistry;
+	}
+	
+	@Override
+	public JmxReporter start(MetricRegistry metricRegistry) {
+		JmxReporter jmxReporter = JmxReporter.forRegistry(metricRegistry).build();
```

Comment:
Should `JmxReporter` not be an instance variable? That way the implementation of `stop()` could just be:

```java
@Override public void stop(MetricRegistry metricRegistry) {
    jmxReporter.stop();
}